hc/accounts/views.py

```diff
@@ -187,6 +188,26 @@ def profile(request):
                                       user=farewell_user).delete()
 
                 messages.info(request, "%s removed from team!" % email)
+        elif "select_allowed_checks" in request.POST:
+            import json
+            post_data = request.POST   # TODO Clean out
+            post_data = json.dumps(dict(post_data))
+            post_data = json.loads(post_data)
+            check_ids = post_data.get('checks', None)
+            member_id = post_data.get('member_id', None)
+
+            user = User.objects.get(pk=member_id[0])
+
+            # remove previous entries
+            for allowed_check in MemberAllowedChecks.objects.filter(user=user):
```

Comment:
We can use relationship spanning here.`for allowed_check in MemberAllowedChecks.objects.filter(user__id=member_id[0])` and remove the need for the extra query to get `user`